src/ucp/wireup/address.c

```diff
@@ -475,6 +475,7 @@ ucp_address_packed_size(ucp_worker_h worker,
     ssize_t value_size;
     const ucp_address_packed_device_t *dev;
     ucp_md_index_t md_index;
+    const ucp_tl_resource_desc_t *tl_rscs;
```

Comment:
minor: rename to rsc (it's a single resource)

Reply:
fixed